packages/clerk-js/src/core/clerk.ts

```diff
@@ -1642,13 +1644,19 @@ export class Clerk implements ClerkInterface {
     });
   };
 
-  #buildUrl = (key: 'signInUrl' | 'signUpUrl', params?: Record<string, string>): string => {
+  #buildUrl = (
+    key: 'signInUrl' | 'signUpUrl',
+    options: RedirectOptions,
+    _initValues?: Record<string, string>,
```

Comment:
🙃 The `_` prefix in parameters is used to mark unused parameters of a function. Are we sure about this naming?